internal/hierarchyconfig/validator.go

```diff
@@ -117,7 +116,7 @@ func (v *Validator) handle(ctx context.Context, log logr.Logger, req *request) a
 	// exists on the K8s server, we need to be able to update its status even though the rest of the
 	// object wouldn't pass legality. We should probably only give the HNC SA the ability to modify
 	// the _status_, though. TODO: https://github.com/kubernetes-sigs/hierarchical-namespaces/issues/80.
-	if isHNCServiceAccount(req.ui) {
+	if webhooks.IsHNCServiceAccount(req.ui) {
```

Comment:
Ditto, move this to the `internal/config` package

Reply:
Moved `IsHNCServiceAccount` to `internal/config/namespace.go` too.